.github/workflows/publish.yml

```diff
@@ -53,4 +75,6 @@ jobs:
           releaseName: "App v__VERSION__"
           body: "See the assets to download this version and install."
           draft: true
-          prerelease: true
+          prerelease: false
+
+    # rust fmt and clippy actions are not included, since we should only release a version that is passed the tests
```

Comment:
Why is this explanation even here? Obviously we don't need them in publish action, because its job is to publish, not to test...

Reply:
> > yaml file changes only take effect when they are merged into main
> 
> I don't think so. Adding and/or removing operating systems definitely takes effect immediately.
> 
> Just like you can see in this PR: CI isn't running at all.

True... Reverting the `yml` ignore changes